### DISCUSSION TITLE
Would love to see an option to make the onscreen keyboard smaller

### DISCUSSION THREAD

**Discussion by nyteshade** [GitSquared/edex-ui#1017] [Ideas]
<img width="3312" alt="Screen Shot 2021-02-01 at 1 55 51 PM" src="https://user-images.githubusercontent.com/225558/106523059-6d349780-6495-11eb-976e-1437af977e3b.png">

**Comment by GitSquared** [comment]
See #832 - the keyboard isn't supposed to overlap like this.
